ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -ex
+
+OPENSTACK_OPERATOR="openstack-operator"
+
+# We don't want to use OpenShift-CI build cluster namespace
+unset NAMESPACE
+
+# PR SHA
+PR_SHA=$(echo ${JOB_SPEC} | jq -r '.refs.pulls[0].sha')
+
+export IMAGE_TAG_BASE=${REGISTRY}/${ORGANIZATION}/${OPENSTACK_OPERATOR}
+export OPENSTACK_OPERATOR_INDEX=${IMAGE_TAG_BASE}-index:${PR_SHA}
+
+if [ ! -d "${HOME}/install_yamls" ]; then
+  cd ${HOME}
+  git clone https://github.com/openstack-k8s-operators/install_yamls.git
+fi
+
+cd ${HOME}/install_yamls
+# Create/enable openstack namespace
+make namespace
+# Creates storage
+# Sometimes it fails to find container-00 inside debug pod
+# TODO: fix issue in install_yamls
+n=0
+retries=3
+while true; do
+  make crc_storage && break
+  n=$((n+1))
+  if (( n >= retries )); then
+    echo "Failed to run 'make crc_storage' target. Aborting"
+    exit 1
+  fi
+  sleep 10
+done
+
+
+# Deploy openstack operator
+make openstack OPENSTACK_IMG=${OPENSTACK_OPERATOR_INDEX}
+# Wait before start checking all deployment status
+# Not expecting to fail here, only in next deployment checks
+n=0
+retries=30
+until [ "$n" -ge "$retries" ]; do
+  oc get deployment openstack-operator-controller-manager && break
+    n=$((n+1))
+    sleep 10
+done
+
+# Check if all deployments are available
+INSTALLED_CSV=$(oc get subscription openstack-operator -o jsonpath='{.status.installedCSV}')
+oc get csv ${INSTALLED_CSV} -o jsonpath='{.spec.install.spec.deployments[*].name}' | \
+timeout ${TIMEOUT_OPERATORS_AVAILABLE} xargs -I {} -d ' ' \
+sh -c 'oc wait --for=condition=Available deployment {} --timeout=-1s'
+
+# Deploy openstack services
+make openstack_deploy
+sleep 60
+
+# Waiting for all services to be ready
+# TODO(dviroel): Add cinder back to status check once ceph backend is functional
+oc get OpenStackControlPlane openstack -o json | \
+jq -r '.status.conditions[] | select(.type | test("^OpenStackControlPlane(?!Cinder)")).type' | \
```

Comment:
+1 on this condition until [1] merges.
When `make ceph` will be available, I guess we can add it to this job and patch the `OpenStackControlPlane` to inject the new Ceph backend to both `Glance` and `Cinder`.

In theory it should be easy enough if we provide an updated `openstackcontrolplane_patched.yaml` file that can just be applied after L60.

[1] https://github.com/openstack-k8s-operators/install_yamls/pull/54

Reply:
@viroel @Sandeepyadav93 now that we have `make ceph` available in `install_yaml`, I'm wondering if we're ready to change the job and enable cinder again